python/pyspark/pandas/frame.py

```diff
@@ -4815,6 +4815,13 @@ def to_spark_io(
         index_col: Optional[Union[str, List[str]]] = None,
         **options
     ) -> None:
+        """An alias for :func:`spark.to_spark_io`.
+        See :meth:`pyspark.pandas.DataFrame.spark.to_spark_io`.
+
+        .. deprecated:: 3.2.0
+            Use :func:`spark.to_spark_io` instead.
```

Comment:
Does this link correctly in the built documentation? I thought it has to be :func:`DataFrame.spark.to_spark_io`

Reply:
@HyukjinKwon Thanks for the review. Updated it.